src/freenas/etc/ix.rc.d/ix-ttys

```diff
@@ -40,13 +40,16 @@ update_ttys()
 			serspeed=""
 		fi
 
+		# Let's make sure that all virtual terminals are set to Pc before we make any changes
+		sed -E -e "s,^(ttyv.*)(freenas|freenas.115200|Pc)(.*)\$,\1Pc\3," /etc/ttys > /etc/ttys.bak && mv /etc/ttys.bak /etc/ttys
```

Comment:
Why mv? Wouldnt sed -i '' have the same effect?

Reply:
It will create a backup file first

Reply:
Both approaches will work. We will need to delete the backup file then if we use -i. What do you think would be the best approach ? I can update accordingly

Reply:
I wont, thats why empty -i, see `man sed`